src/features/ShowHelp.ts

```diff
@@ -4,6 +4,7 @@
 import vscode = require("vscode");
 import { NotificationType } from "vscode-languageclient";
 import { LanguageClientConsumer } from "../languageClientConsumer";
+import { LanguageClient } from "vscode-languageclient/node";
```

Comment:
Should be `type LanguageClient` to be clear you're only using this for type validation and not actually instantiating it

Reply:
TIL